### PR TITLE
drop support for .NET Core 2.1

### PR DESCRIPTION
**Pull request by filipw**
.NET Core 2.1 reached end of life 2 days ago https://dotnet.microsoft.com/platform/support/policy/dotnet-core

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,7 +313,7 @@ To consume a script package all we need to do specify the NuGet package in the `
 The following example loads the [simple-targets](https://www.nuget.org/packages/simple-targets-csx) package that contains script files to be included in our script.
 
 ```C#
-#! "netcoreapp2.1"
+#! "netcoreapp3.1"
```
Comment (seesharper) [inline README.md:316]: This is not used anymore so it should probably be removed from the docs

https://github.com/filipw/dotnet-script/commit/a10f53a99c8be51a0dcf5d3d7b0461d7d339649f

**Review by seesharper — APPROVED** [review]
Looks good. One less thing to worry about. I had just one comment on the shebang

File: README.md (hunk 1 above)
Comment (filipw) [reply README.md]: thanks I forgot about that

Comment (seesharper) [reply README.md]: 👍

**Pull request merged**